doc/API.md

```diff
@@ -0,0 +1,212 @@
+# Minisafe API
+
+`minisafe` exposes a [JSON-RPC 2.0](https://www.jsonrpc.org/specification)
+interface over a Unix Domain socket.
+
+Note that all addresses are bech32-encoded *version 0* native Segwit `scriptPubKey`s.
+
+| Command                                                     | Description                                          |
+| ----------------------------------------------------------- | ---------------------------------------------------- |
+| [`help`](#help)                                             | Display all available commands                       |
+| [`stop`](#stop)                                             | Stops the minisafe daemon                            |
+| [`getinfo`](#getinfo)                                       | Display general information                          |
+| [`getaddress`](#getaddress)                                 | Get an address                                       |
+| [`listutxos`](#listutxos)                                   | Display a paginated list of utxos                    |
+| [`getspendtx`](#getspendtx)                                 | Retrieve the Minisafe spend transaction to sign      |
+| [`updatespendtx`](#updatespendtx)                           | Update the Minisafe spend transaction                |
+| [`deletespendtx`](#deletespendtx)                           | Delete the Minisafe spend transaction                |
+| [`listspendtxs`](#listspendtxs)                             | Retrieve the Minisafe pending outgoing transactions  |
```

Comment:
I agree it makes sense to have the same concept of pending Spend txs as in revaultd. Would be worth discussing it with @kloaec too.

Reply:
Also, let's name it with `psbt` rather than `spendtx`?